telemetry.py

```diff
@@ -0,0 +1,158 @@
+import os
+import time
+from datetime import datetime, UTC
+import threading
+from queue import Queue
+
+from prometheus_client import start_http_server, Gauge, Enum, Counter, Histogram
+import psutil
+
+from mission import Telemetry
+
+
+class KSPTelemetry:
+    """
+    A class to handle publishing metrics periodically
```

Comment:
given results of POC of https://github.com/cazlo/ksp-krpc-rocket-avionics/pull/2

thinking to refactor this to abstract at least read telemetry calls to kRPC (automagically?) using stream API.
also abstracting write calls to kRPC affords additional opportunity to telemeter output from the control system.
some insights we can get from instrumenting control system output includes measurement of input lag, potential mistakes during control system dev, etc.

munging up these 2 concerns allows us to publish telem we use as input into the avionics system in a cross cutting concern, with little/no effort needed in avionics scripts to setup the metrics system (i.e. could remove most or all of the 'publish throttle telem after update`, `register metric for publish` stuff done during script init)

Reply:
if this ends up getting done in this PR, make sure to add a diagram showing the threading model and how it is used to reduce impact of instrumentation on the code itself